### PR TITLE
Completed dom-querying exercise

### PR DESCRIPTION
**Pull request by wang-morris**
Screen Shot 2022-06-20 at 7 08 38 PM

### CODE REVIEW

File: dom-querying/main.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+console.log('hello, world');
+
+console.log(document.querySelector('h1'));
+console.dir('h1');
```
Comment (BrettAlbright) [inline dom-querying/main.js]: this is not what you want. You need to pass a DOM element to dir. I recommend storing the queryselected element in a variable so you can pass that variable to both the log and dir method calls.

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
make sure that you are passing an element to `console.dir` not a selector

**Comment by wang-morris** [comment]
Screen Shot 2022-06-20 at 7 27 53 PM

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**